runtime/lua/vim/lsp.lua

```diff
@@ -2270,9 +2270,9 @@ end
 
 --- Implements 'omnifunc' compatible LSP completion.
 ---
----@see |complete-functions|
----@see |complete-items|
----@see |CompleteDone|
+---@see help `|complete-functions|`
+---@see help `|complete-items|`
+---@see help `|CompleteDone|`
```

Comment:
?? why are these taglinks being converted to codespans? If LSP is complaining about that, just ignore it. The generated docs should have taglinks, not codespans.

Reply:
luals need a name for `@see` we missing this name.  so many annoying warns for this.

Reply:
We could instead disable it in the config:

```diff
diff --git a/contrib/luarc.json b/contrib/luarc.json
index ebad0581b..31126e421 100644
--- a/contrib/luarc.json
+++ b/contrib/luarc.json
@@ -11,10 +11,13 @@
           "after_each",
           "setup",
           "teardown",
           "finally",
           "lfs"
+        ],
+        "disable": [
+          "luadoc-miss-see-name"
         ]
     },
     "workspace": {
         "library": [
             "runtime/lua",
```

Reply:
thanks this make sense!